### PR TITLE
[zh] translate SIG-Usability-Spotlight to chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/36346

https://kubernetes.io/blog/2021/07/15/sig-usability-spotlight-2021/ does not have chinese version

so need to translate it into Chinese

### CODE REVIEW

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+## 介绍
+
+<!--
+Are you interested in learning about what [SIG Usability](https://github.com/kubernetes/community/tree/master/sig-usability) does 
+and how you can get involved? Well, you're at the right place.
+SIG Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community.
+In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab),
+who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.
+-->
+你是否有兴趣了解 SIG 可用性以及如何参与？那你来对地方了。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:31]:
```suggestion
你是否有兴趣了解 [SIG Usability](https://github.com/kubernetes/community/tree/master/sig-usability) 做什么？你是否想知道如何参与？那你来对地方了。
```
The link is missing.

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+SIG 可用性旨在让新人更容易使用 Kubernetes，其主要活动是为社区进行用户研究。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:32]:
```suggestion
SIG Usability 旨在让 Kubernetes 更易于触达新用户，其主要活动是对社区实施用户调研。
```

**Review by windsonsea — COMMENTED** [review]
Preview FYI: 
https://deploy-preview-36349--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2021/07/15/sig-usability-spotlight-2021/

Need not to localize a SIG name like SIG Usability.

**Comment by twilight0620** [comment]
I have modified them. Thanks for reviewing

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+---
+layout: blog
+title: "聚焦 SIG Usability "
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:3]:
```suggestion
title: "聚焦 SIG Usability"
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+such as incorporating internationalization and ensuring the openness of documentation.
+-->
+这涵盖了用户体验和可访问性等方面。
+SIG 的目标是确保 Kubernetes 项目能够最大限度地被具有广泛基础和能力的人使用，
+例如纳入国际化和确保文档的开放性。
+
+<!--
+### Q. Why should new and existing contributors consider joining SIG Usability?
+-->
+### 问：为什么新的和现有的贡献者应该考虑加入 SIG Usability ？
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:77]:
```suggestion
### 问：为什么新的和现有的贡献者应该考虑加入 SIG Usability？
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+<!--
+- User research projects, where people can help understand the usability of the end-user experiences, including error messages, end-to-end tasks, etc.
+-->
+- 用户研究项目可以让人们帮助了解最终用户体验的可用性，包括错误消息、端到端任务等。
+<!--
+- Accessibility guidelines for Kubernetes community artifacts, examples include: 
+internationalization of documentation, color choices for people with color blindness, ensuring compatibility with screen reader technology,
+ user interface design for core components with user interfaces, and more.
+-->
+- Kubernetes 社区组件的可访问性指南，包括：文档的国际化、色盲人群的颜色选择、确保与屏幕阅读器技术的兼容性、具有用户界面的核心组件的用户界面设计等等。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:92]:
```suggestion
- Kubernetes 社区组件的可访问性指南，包括：文档的国际化、色盲人群的颜色选择、
  确保与屏幕阅读器技术的兼容性、核心 UI 组件的用户界面设计等等。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+
+<!--
+### Q. What do you do to help new contributors get started?
+-->
+### 问：如何帮助新的贡献者入门？
+
+<!--
+A. New contributors can get started by shadowing one of the user interviews, going through user interview transcripts, analyzing them, and designing surveys.
+-->
+答：新的贡献者可以通过跟踪其中一个用户访谈、浏览用户访谈记录、分析它们和设计调查来开始。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:102]:
```suggestion
答：新的贡献者们刚开始可以旁观参与其中一个用户访谈，浏览用户访谈记录，分析这些记录并设计调查过程。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+SIG Usability is also open to new project ideas. 
+If you have an idea, we’ll do what we can to support it. 
+There are regular SIG Meetings where people can ask their questions live. 
+These meetings are also recorded for those who may not be able to attend.
+ As always, you can reach out to us on Slack as well.
+-->
+SIG Usability 也对新的项目想法持开放态度。
+如果你有想法，我们将尽我们所能支持它。
+有定期的 SIG 会议，人们可以现场提问。
+这些会议也会为那些可能无法参加的人录制。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:114]:
```suggestion
这些会议也会录制会议视频，方便那些可能无法参会的人。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+such as trends in learning to deploy a new system, error messages they receive, and workflows.
+-->
+答：简单来说，该调查收集了人们如何使用 Kubernetes 的信息，
+例如学习部署新系统的趋势、他们收到的错误消息和工作流程。
+
+<!--
+One of our goals is to standardize the responses accordingly.
+The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
+-->
+我们的目标之一是标准化相应的响应。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:133]:
```suggestion
我们的目标之一是形成相应的标准化应答。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+最终目标是分析那些需求没有得到满足的重要用户故事的调查响应。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:134]:
```suggestion
最终目标是分析那些需求没有得到满足的重要用户故事的调查应答。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+-->
+### 问：你有什么特别想招聘的技能吗？SIG Usability 的贡献者可能要学习哪些技能？
+
+<!--
+A. Although contributing to SIG Usability does not have any pre-requisites as such, 
+experience with user research, qualitative research, or prior experience with how to conduct an interview would be great plus points. 
+Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.
+-->
+答：虽然为 SIG Usability 做贡献没有任何先决条件，
+但用户研究、定性研究的经验或之前如何进行面试的经验将是很好的加分项。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:147]:
```suggestion
但用户研究、定性研究的经验或之前如何进行访谈的经验将是很好的加分项。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+One of our goals is to standardize the responses accordingly.
+The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
+-->
+我们的目标之一是标准化相应的响应。
+最终目标是分析那些需求没有得到满足的重要用户故事的调查响应。
+
+<!--
+### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG Usability likely to learn?
+-->
+### 问：你有什么特别想招聘的技能吗？SIG Usability 的贡献者可能要学习哪些技能？
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:139]:
```suggestion
### 问：招募贡献者时你希望他们具备什么特别的技能吗？SIG Usability 的贡献者可能要学习哪些技能？
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.
+-->
+答：虽然为 SIG Usability 做贡献没有任何先决条件，
+但用户研究、定性研究的经验或之前如何进行访谈的经验将是很好的加分项。
+定量研究，如调查设计和筛选，也很有帮助，也是我们希望贡献者学习的东西。
+
+<!--
+### Q. What are you getting positive feedback on, and what’s coming up next for SIG Usability?
+-->
+### 问：您在哪些方面获得了积极的反馈，以及 SIG Usability 接下来会发生什么？
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:154]:
```suggestion
### 问：你在哪些方面获得了积极的反馈，以及 SIG Usability 接下来会发生什么？
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+Currently, we are focused on finishing the study mentioned in our [talk](https://www.youtube.com/watch?v=Byn0N_ZstE0), 
+also our project for this year. We are always happy to have new contributors join us.
+-->
+目前，我们正专注于完成我们[谈话](https://www.youtube.com/watch?v=Byn0N_ZstE0)中提到的调研，
+也是我们今年的项目。我们总是很高兴有新的贡献者加入我们。
+
+<!--
+### Q: Any closing thoughts/resources you’d like to share?
+-->
+### 问：您想分享任何结束的想法/资源吗？
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:173]:
```suggestion
### 问：你想分享任何结束的想法/资源吗？
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+We have also had a lot of people reach out to us via Slack showcasing their interest in the SIG.
+-->
+答：我们有新成员加入并定期参加月度会议，并表现出对成为贡献者和帮助社区的兴趣。
+我们也有很多人通过 Slack 与我们联系，展示他们对 SIG 的兴趣。
+
+<!--
+Currently, we are focused on finishing the study mentioned in our [talk](https://www.youtube.com/watch?v=Byn0N_ZstE0), 
+also our project for this year. We are always happy to have new contributors join us.
+-->
+目前，我们正专注于完成我们[谈话](https://www.youtube.com/watch?v=Byn0N_ZstE0)中提到的调研，
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:167]:
```suggestion
目前，我们正专注于完成我们[演講](https://www.youtube.com/watch?v=Byn0N_ZstE0)中提到的调研，
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md (hunk 12 above)
Comment (twilight0620) [reply content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:154]: 访谈的提问可以用您吧？

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 15]
```diff
@@ -0,0 +1,212 @@
...
+答：SIG Usability 的高层次开始是因为 Kubernetes 没有专门的用户体验团队。
+SIG Usability 的范围集中在 Kubernetes 项目的终端客户易用性上。
+主要活动是社区的用户研究，其中包括与 Kubernetes 用户交谈。
+
+<!--
+This covers points like user experience and accessibility. 
+The objectives of the SIG are to guarantee that the Kubernetes project is maximally usable by people of a wide range of foundations and capacities,
+such as incorporating internationalization and ensuring the openness of documentation.
+-->
+这涵盖了用户体验和可访问性等方面。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:70]:
```suggestion
所涉及的包括用户体验和可访问性等方面。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 16]
```diff
@@ -0,0 +1,212 @@
...
+SIG 的目标是确保 Kubernetes 项目能够最大限度地被具有广泛基础和能力的人使用，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:71]:
```suggestion
SIG 的目标是确保 Kubernetes 项目能够最大限度地被具有各类不同基础和能力的人使用，
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 17]
```diff
@@ -0,0 +1,212 @@
...
+例如纳入国际化和确保文档的开放性。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:72]:
```suggestion
例如引入文档的国际化并确保其开放性。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 18]
```diff
@@ -0,0 +1,212 @@
...
+
+<!--
+### Q. Why should new and existing contributors consider joining SIG Usability?
+-->
+### 问：为什么新的和现有的贡献者应该考虑加入 SIG Usability？
+
+<!--
+A. There are plenty of territories where new contributors can begin. For example:
+-->
+答：新的贡献者可以从很多领域开始。例如：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:82]:
```suggestion
答：新的贡献者可以在很多领域着手。例如：

```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 19]
```diff
@@ -0,0 +1,212 @@
...
+- User research projects, where people can help understand the usability of the end-user experiences, including error messages, end-to-end tasks, etc.
+-->
+- 用户研究项目可以让人们帮助了解最终用户体验的可用性，包括错误消息、端到端任务等。
+<!--
+- Accessibility guidelines for Kubernetes community artifacts, examples include: 
+internationalization of documentation, color choices for people with color blindness, ensuring compatibility with screen reader technology,
+ user interface design for core components with user interfaces, and more.
+-->
+- Kubernetes 社区组件的可访问性指南，包括：文档的国际化、色盲人群的颜色选择、
+确保与屏幕阅读器技术的兼容性、核心 UI 组件的用户界面设计等等。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:93]:
```suggestion
  确保与屏幕阅读器技术的兼容性、核心 UI 组件的用户界面设计等等。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 20]
```diff
@@ -0,0 +1,212 @@
...
+<!--
+SIG Usability is also open to new project ideas. 
+If you have an idea, we’ll do what we can to support it. 
+There are regular SIG Meetings where people can ask their questions live. 
+These meetings are also recorded for those who may not be able to attend.
+ As always, you can reach out to us on Slack as well.
+-->
+SIG Usability 也对新的项目想法持开放态度。
+如果你有想法，我们将尽我们所能支持它。
+有定期的 SIG 会议，人们可以现场提问。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:114]:
```suggestion
我们有定期的 SIG 会议，人们可以现场提问。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 21]
```diff
@@ -0,0 +1,212 @@
...
+<!--
+### Q. What does the survey include?
+-->
+### 问：调查包括什么？
+
+<!--
+A. In simple terms, the survey gathers information about how people use Kubernetes, 
+such as trends in learning to deploy a new system, error messages they receive, and workflows.
+-->
+答：简单来说，该调查收集了人们如何使用 Kubernetes 的信息，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:127]:
```suggestion
答：简单来说，调查会收集人们如何使用 Kubernetes 的信息，
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 22]
```diff
@@ -0,0 +1,212 @@
...
+例如学习部署新系统的趋势、他们收到的错误消息和工作流程。
+
+<!--
+One of our goals is to standardize the responses accordingly.
+The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
+-->
+我们的目标之一是形成相应的标准化应答。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:134]:
```suggestion
我们的目标之一是根据需要对反馈进行标准化。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 23]
```diff
@@ -0,0 +1,212 @@
...
+最终目标是分析那些需求没有得到满足的重要用户故事的调查应答。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:135]:
```suggestion
最终目标是分析那些需求没有得到满足的重要用户故事的调查反馈。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 24]
```diff
@@ -0,0 +1,212 @@
...
+<!--
+### Q. What are you getting positive feedback on, and what’s coming up next for SIG Usability?
+-->
+### 问：您在哪些方面获得了积极的反馈，以及 SIG Usability 接下来会发生什么？
+
+<!--
+A. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. 
+We have also had a lot of people reach out to us via Slack showcasing their interest in the SIG.
+-->
+答：我们有新成员加入并定期参加月度会议，并表现出对成为贡献者和帮助社区的兴趣。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:160]:
```suggestion
答：我们一直有新成员加入并经常参加月度会议，并表现出对成为贡献者和帮助社区的兴趣。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 25]
```diff
@@ -0,0 +1,212 @@
...
+我们也有很多人通过 Slack 与我们联系，展示他们对 SIG 的兴趣。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:161]:
```suggestion
我们也有很多人通过 Slack 与我们联系，表达他们对 SIG 的兴趣。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md (hunk 14 above)
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:167]: Sea-n 的建议是正确的，为什么没有改，并且直接将反馈标记为 resolved？

```suggestion
目前，我们正专注于完成我们[演讲](https://www.youtube.com/watch?v=Byn0N_ZstE0)中提到的调研，
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md (hunk 13 above)
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:173]:
```suggestion
### 问：在结束之前，你还有什么想法/资源要分享吗？
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 26]
```diff
@@ -0,0 +1,212 @@
...
+### Q: Any closing thoughts/resources you’d like to share?
+-->
+### 问：您想分享任何结束的想法/资源吗？
+
+<!--
+A. We love meeting new contributors and assisting them in investigating different Kubernetes project spaces. 
+We will work with and team up with other SIGs to facilitate engaging with end-users, running studies, 
+and help them integrate accessible design practices into their development practices.
+-->
+答：我们喜欢结识新的贡献者并帮助他们研究不同的 Kubernetes 项目空间。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:180]:
```suggestion
答：我们喜欢结识新的贡献者并帮助他们研究不同的 Kubernetes 项目领域。
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 27]
```diff
@@ -0,0 +1,212 @@
...
+Here are some resources for you to get started:
+- [GitHub](https://github.com/kubernetes/community/tree/master/sig-usability)
+- [Mailing list](https://groups.google.com/g/kubernetes-sig-usability)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fusability)
+- [Slack](https://slack.k8s.io/)
+- [Slack channel #sig-usability](https://kubernetes.slack.com/archives/CLC5EF63T)
+-->
+这里有一些资源供你入门：
+- [GitHub](https://github.com/kubernetes/community/tree/master/sig-usability)
+- [Mailing list](https://groups.google.com/g/kubernetes-sig-usability)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:193]:
```suggestion
- [邮件列表](https://groups.google.com/g/kubernetes-sig-usability)
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md [hunk 28]
```diff
@@ -0,0 +1,212 @@
...
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fusability)
+- [Slack](https://slack.k8s.io/)
+- [Slack channel #sig-usability](https://kubernetes.slack.com/archives/CLC5EF63T)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:196]:
```suggestion
- [Slack 频道 #sig-usability](https://kubernetes.slack.com/archives/CLC5EF63T)
```

File: content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md (hunk 12 above)
Comment (ydFu) [reply content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:154]: > 访谈的提问可以用您吧？

https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E7%89%B9%E6%AE%8A%E8%AF%8D%E6%B1%87

Comment (tengqm) [reply content/zh-cn/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md:154]: 好问题，我认为可以。

**Pull request merged**